src/java.base/share/classes/java/util/jar/JarOutputStream.java

```diff
@@ -78,6 +78,11 @@ public JarOutputStream(OutputStream out) throws IOException {
      * to the start of the entry data. This method will also close
      * any previous entry. The default compression method will be
      * used if no compression method was specified for the entry.
+     * When writing a compressed (DEFLATED) entry, and the compressed
+     * size has not been explicitly set with the {@link
+     * ZipEntry#setCompressedSize(long)} method, then the compressed
+     * size will be set to the actual compressed size after deflation.
+     * <p>
```

Comment:
I'm happy with the wording. What you think about put a p tag before "The default compression method ..." so that it goes into its own paragraph? I'm asking because it's inconsistent to have the first paragraph include the details on the compression method and the details on the current time pushed into a second paragraph - if you generate the javadoc then you'll see what I mean.

Reply:
I think that is a good idea to at the paragraph tag as you suggest

Reply:
Done